src/Pages/ComparisonGraph.cshtml.cs

```diff
@@ -56,9 +59,22 @@ public async Task<IActionResult> OnGetFilter(DateTime startDate, DateTime endDat
 
         private async Task SetData()
         {
-            UserId = int.Parse(User.FindFirst(ClaimTypes.Name).Value);
-            Expenses = (await EntryDatabase.ReadListAsync(UserId, EntryType.Expense)).GetFrom(StartDate).GetTo(EndDate).GetSum();
-            Income = (await EntryDatabase.ReadListAsync(UserId, EntryType.Income)).GetFrom(StartDate).GetTo(EndDate).GetSum();
+            try
+            {
+                UserId = int.Parse(User.FindFirst(ClaimTypes.Name).Value);
+                Expenses = (await EntryDatabase.ReadListAsync(UserId, EntryType.Expense)).GetFrom(StartDate)
+                    .GetTo(EndDate).GetSum();
+                Income = (await EntryDatabase.ReadListAsync(UserId, EntryType.Income)).GetFrom(StartDate).GetTo(EndDate)
+                    .GetSum();
+            }
+            catch (Exception ex)
+            {
+                _logger.LogInformation(ex.ToString());
+                WasException = true;
+                Expenses = 1000;
+                Income = 1000;
```

Comment:
I'm not sure about using lists since it would be more time costly and in this page separate entries does not matter that much. Though I would be happy to use random numbers in range 500-5000 here, if time allows me :D

Reply:
Makes sense :D